main/realmain.cc

```diff
@@ -1,22 +1,28 @@
-// has to be included first as it violates our poisons
-#include "core/proto/proto.h"
-
+#ifdef SORBET_REALMAIN_MIN
+// minimal build to speedup compilation. Remove extra features
+#define FULL_BUILD_ONLY(X) false; // do nothing
+#else
+#define FULL_BUILD_ONLY(X) X;
```

Comment:
If this is not unused, should the definition be like this?

```suggestion
#define FULL_BUILD_ONLY(X) (X);
```